mantle/platform/qemu.go

```diff
@@ -771,6 +771,12 @@ func (builder *QemuBuilder) addDiskImpl(disk *Disk, primary bool) error {
 				pId, disk.attachEndPoint))
 		}
 	} else {
+		if !disk.NbdDisk {
+			// In the non-multipath/nbd case we can just unlink the disk now
+			// and avoid leaking space if we get Ctrl-C'd.
+			os.Remove(disk.dstFileName)
+		}
+		disk.dstFileName = ""
```

Comment:
```suggestion
		if !disk.NbdDisk {
			// In the non-multipath/nbd case we can just unlink the disk now
			// and avoid leaking space if we get Ctrl-C'd.
			os.Remove(disk.dstFileName)
			// and unset so it doesn't get added to the list of tmpFiles in
			// `Exec()`
			disk.dstFileName = ""
		}
```

Otherwise I think in the nbd path we end up not cleaning the disk and socket?

Reply:
OK yeah, I'm just going to stick everything in a single tempdir.